.flake8

```diff
@@ -1,2 +1,2 @@
 [flake8]
-ignore = E501,W504
+ignore = E501,W504,E402
```

Comment:
Is this to prevent a linting error on the tests? The CI checks don't run linting on the test code (although maybe it should). Can you undo this change.

Reply:
done